src/Cerbere.vala

```diff
@@ -68,15 +68,16 @@ public class Cerbere.App : Application {
         if (sm_client != null) {
             // The session manager may ask us to quit the service, and so we do.
             sm_client.stop_service.connect (quit_service);
-            // Cleanly shutdown when receiving SIGTERM.
+            // Cleanly shutdown when receiving SIGTERM or SIGHUP.
             Posix.signal (Posix.Signal.TERM, handle_sigterm);
+            Posix.signal (Posix.Signal.TERM, handle_sighup);
```

Comment:
shouldn't this be `HUP` instead of `TERM` ? copy-paste mistake?

Reply:
Yes, definitely should be HUP. I think merging went wrong at some point.
Amended now, PTAL.